public-src/phphants/scss/style.scss

```diff
@@ -264,16 +264,20 @@ header .login-status-text {
     cursor: pointer;
 }
 
-.content{
-    flex: 1 0 auto;
+.content {
+    display: flex;
```

Comment:
Hmm. This says support is ok...
![screen shot 2017-06-12 at 20 18 42](https://user-images.githubusercontent.com/2809847/27051012-7651cd1c-4fac-11e7-818c-3d6cbe21d881.png)

I don't have a windows machine handy to test on, so I'll have to put this on hold